### PR TITLE
Update get_started.md

### PR DESCRIPTION
**Pull request by irexyc**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/get_started.md [hunk 1]
```diff
@@ -115,46 +114,44 @@ The supported platform and device matrix is presented as following:
...
-wget https://github.com/open-mmlab/mmdeploy/releases/download/v1.0.0rc3/mmdeploy-1.0.0rc3-linux-x86_64-cuda11.1-tensorrt8.2.3.0.tar.gz
-tar -zxvf mmdeploy-1.0.0rc3-linux-x86_64-cuda11.1-tensorrt8.2.3.0.tar.gz
-cd mmdeploy-1.0.0rc3-linux-x86_64-cuda11.1-tensorrt8.2.3.0
-pip install dist/mmdeploy-1.0.0rc3-py3-none-linux_x86_64.whl
-pip install sdk/python/mmdeploy_runtime-1.0.0rc3-cp38-none-linux_x86_64.whl
-cd ..
-# install inference engine: TensorRT
-# !!! Download TensorRT-8.2.3.0 CUDA 11.x tar package from NVIDIA, and extract it to the current directory
+# 1. install MMDeploy model converter
+pip install mmdeploy==1.0.0rc3
```
Comment (RunningLeon) [inline docs/en/get_started.md:121]: new version would be 1.0.0

File: .github/workflows/prebuild.yml [hunk 2]
```diff
@@ -7,6 +7,9 @@ on:
       - dev-1.x
     paths:
       - "mmdeploy/version.py"
+  pull_request:
```
Comment (RunningLeon) [inline .github/workflows/prebuild.yml:10]: Should be deleted before merge.

File: docs/zh_cn/get_started.md [hunk 3]
```diff
@@ -226,12 +223,15 @@ result = inference_model(
 你可以直接运行预编译包中的 demo 程序，输入 SDK Model 和图像，进行推理，并查看推理结果。
 
 ```shell
-cd mmdeploy-1.0.0rc3-linux-x86_64-cuda11.1-tensorrt8.2.3.0
+wget https://github.com/open-mmlab/mmdeploy/releases/download/v1.0.0rc3/mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3.tar.gz
+tar xf mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3
```
Comment (RunningLeon) [inline docs/zh_cn/get_started.md:227]: mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3.tar.gz?

File: docs/en/get_started.md [hunk 4]
```diff
@@ -233,12 +230,14 @@ result = inference_model(
 You can directly run MMDeploy demo programs in the precompiled package to get inference results.
 
 ```shell
-cd mmdeploy-1.0.0rc3-linux-x86_64-cuda11.1-tensorrt8.2.3.0
+wget https://github.com/open-mmlab/mmdeploy/releases/download/v1.0.0rc3/mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3.tar.gz
+tar xf mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3
```
Comment (RunningLeon) [inline docs/en/get_started.md:234]: `mmdeploy-1.0.0rc3-linux-x86_64-cuda11.3.tar.gz` ?

File: docs/zh_cn/get_started.md (hunk 3 above)
Comment (irexyc) [reply docs/zh_cn/get_started.md:227]: do you mean the tensorrt suffix? I remove it as this package support both tensorrt and onnxruntime-gpu inference.

Comment (RunningLeon) [reply docs/zh_cn/get_started.md:227]: OK

**Review by lvhan028 — APPROVED** [review]

File: docs/en/get_started.md [hunk 5]
```diff
@@ -115,46 +114,44 @@ The supported platform and device matrix is presented as following:
...
-# !!! Download TensorRT-8.2.3.0 CUDA 11.x tar package from NVIDIA, and extract it to the current directory
+# 1. install MMDeploy model converter
+pip install mmdeploy==1.0.0rc3
+
+# 2. install MMDeploy sdk inference
+# you can install one to install according whether you need gpu inference
+# 2.1 support onnxruntime
+pip install mmdeploy-runtime==1.0.0rc3
+# 2.2 support onnxruntime-gpu, tensorrt
+pip install mmdeploy-runtime-cuda==1.0.0rc3
```
Comment (RunningLeon) [inline docs/en/get_started.md:128]: is here `mmdeploy-runtime-cpu`?

File: docs/zh_cn/get_started.md [hunk 6]
```diff
@@ -109,46 +109,43 @@ mim install "mmcv>=2.0.0rc2"
...
-# !!! 从 NVIDIA 官网下载 TensorRT-8.2.3.0 CUDA 11.x 安装包并解压到当前目录
+# 1. 安装 MMDeploy 模型转换工具（含trt/ort自定义算子）
+pip install mmdeploy==1.0.0rc3
+
+# 2. 安装 MMDeploy SDK推理工具
+# 根据是否需要GPU推理可任选其一进行下载安装
+# 2.1 支持 onnxruntime 推理
+pip install mmdeploy-runtime==1.0.0rc3
+# 2.2 支持 onnxruntime-gpu tensorrt 推理
+pip install mmdeploy-runtime-cuda==1.0.0rc3
```
Comment (RunningLeon) [inline docs/zh_cn/get_started.md:123]: is here `mmdeploy-runtime-cpu`?

**Pull request merged**